core/src/main/java/io/cucumber/core/model/FeaturePath.java

```diff
@@ -43,12 +47,17 @@ public static URI parse(String featureIdentifier) {
     private static URI parseProbableURI(String featureIdentifier) {
         return URI.create(featureIdentifier);
     }
-
+    
+    private static boolean isWindowsOS() { return System.getProperty("os.name").contains("Windows");}
```

Comment:
I believe the return value of `System.getProperty("os.name")` should be normalized first using:

```
    private static String normalize(String value) {
        if (value == null) {
            return "";
        }
        return value.toLowerCase(Locale.US).replaceAll("[^a-z0-9]+", "");
}
```

Reply:
Good point!  Updated code to normalise as recommended.